README.md

```diff
@@ -9,632 +11,57 @@
 
 It uses API functions of Media Server for Remoting and Streaming [Kurento](https://www.kurento.org)).
 
-Getting Started
-===============
-Please visit the project website for the latest information:
-  - https://openmeetings.apache.org/
-
-Documentation for Installation and Upgrade:
-  - https://openmeetings.apache.org/installation.html
-  - https://openmeetings.apache.org/Upgrade.html
-
-Along with the developer mailing list archive:
-  - https://openmeetings.apache.org/mailing-lists.html
-
-
-System Requirements
-===================
-You need a platform that supports [Java SE 17](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html).
-
-
-Building and running
-====================
-To build from source code:
-
-  - Requirements:
-    Sources compilation require `Java SE 17`, [Apache Maven 3.8.7 or later](https://maven.apache.org/).
-  - To build the OpenMeetings project run mvn in the root directory:
-     `mvn clean install -PallModules`
-  - To run the OpenMeetings project:
-    - go to `openmeetings-server/target` directory
-    - extract `apache-openmeetings-x.x.x.tar.gz` (or `apache-openmeetings-x.x.x.zip` for windows)
-    - enter `apache-openmeetings-x.x.x` directory and execute `./bin/startup.sh` (`./bin/startup.bat` for Windows)
-
-For a detailed documentation on how to build from source and options see:
-    https://openmeetings.apache.org/BuildInstructions.html
-
-Builds and CI
-====================
-
-| Description | Jenkins CI |
-| :--- | :---: |
-| Master nightly | [![Build Status](https://ci-builds.apache.org/job/OpenMeetings/job/openmeetings/badge/icon)](https://ci-builds.apache.org/job/OpenMeetings/job/openmeetings/) |
+### Getting Started
+For the latest information, please visit the project website:
+  - [OpenMeetings Website](https://openmeetings.apache.org/)
+
+Documentation for installation and upgrade can be found at:
+  - [Installation Guide](https://openmeetings.apache.org/installation.html)
+  - [Upgrade Instructions](https://openmeetings.apache.org/Upgrade.html)
+
+Developer mailing list archive:
+  - [Mailing List](https://openmeetings.apache.org/mailing-lists.html)
+
+### Prerequisites
+Apache OpenMeetings requires the following software to run:
+- [Java SE 17](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html)
+- [Apache Maven 3.8.7 or later](https://maven.apache.org/)
+
+### System Requirements
+You need a platform that supports Java SE 17.
+
+### Installation Steps
```

Comment:
The below instructions are for building from sources and running, installation instructions are at the separate wiki page